tailwind.config.js

```diff
@@ -76,81 +75,94 @@ module.exports = {
 			0: '0',
 			px: '1px',
 			pico: '1rem',
-			nano: '3rem',
+			nano: '4rem',
 			micro: '8rem',
 			thinnest: '12rem',
 			thinner: '16rem',
 			thin: '20rem',
+			'thin+': '24rem',
 			broad: '32rem',
 			broader: '40rem',
 			broadest: '48rem',
-			large: '72rem',
-			larger: '90rem',
-			largest: '140rem',
+			'broadest+': '64rem',
+			large: '82rem',
+			larger: '100rem',
+			largest: '130rem',
+			'largest+': '170rem',
 			mega: '220rem',
+			'mega+': '335rem',
 			giga: '375rem',
 			tera: '480rem',
+			'tera+': '600rem',
 			tiniest: '640rem',
 			tinier: '720rem',
 			tiny: '790rem',
 			smallest: '880rem',
-			smaller: '1200rem',
+			smaller: '1260rem',
 			small: '1300rem',
 			big: '1450rem',
 			bigger: '1600rem',
 			biggest: '1800rem',
-			'-pico': '-1rem',
-			'-nano': '-3rem',
-			'-micro': '-5rem',
-			'-thinnest': '-10rem',
-			'-thinner': '-16rem',
-			'-thin': '-20rem',
-			'-thin+': '-30rem',
-			'-broad': '-40rem',
-			'-broader': '-60rem',
-			'-broadest': '-80rem',
-			'-large': '-100rem',
-			'-larger': '-120rem',
-			'-largest': '-180rem',
-			'-mega': '-220rem',
-			'-giga': '-375rem',
-			'-tera': '-480rem',
-			'-tiniest': '-640rem',
-			'-tinier': '-720rem',
-			'-tiny': '-790rem',
-			'-smallest': '-946rem',
-			'-smaller': '-1200rem',
-			'-small': '-1300rem',
-			'-big': '-1450rem',
-			'-bigger': '-1600rem',
-			'-biggest': '-1800rem'
+			'1/2': '50%',
+			'1/3': 'calc(100% / 3 * 1)',
+			'2/3': 'calc(100% / 3 * 2)',
+			'1/4': '25%',
+			'3/4': '75%',
+			'1/5': '20%',
+			'2/5': '40%',
+			'3/5': '60%',
+			'4/5': '80%',
+			'1/10': '10%',
+			'3/10': '30%',
+			'7/10': '70%',
+			'9/10': '90%',
+			'1/20': '5%',
+			'3/20': '15%',
+			'7/20': '35%',
+			'9/20': '45%',
+			'11/20': '55%',
+			'13/20': '65%',
+			'17/20': '85%',
+			'19/20': '95%',
+			full: '100%',
+			'screen-1/4': '25vh',
```

Comment:
The choice of vh for "screen" here is arbitrary. We could need vw or vmin. Should probably be `'1/4vh' : '25vh'` and then could also have `'1/4vw' : '25vw'`

Reply:
Yes, already removed it in the last commit.